core/classes/Core/User.php

```diff
@@ -229,23 +229,6 @@ public function getGroupStyle(): string {
         return $css;
     }
 
-    /**
-     * Get the logged in user's IP address.
-     *
-     * @return string Their IP.
-     */
-    public function getIP(): string {
```

Comment:
@Derkades what would you think about keeping this method and just forwarding it to `Util::getRemoteIp()`? `$user->getIP()` reads more fluently 🤔

Reply:
But it has nothing to do with the User object, instead with the current connection. Someone might do
```
$user = new User(4)
$user->getIP()
```
And expect to get the IP address for user 4, which is not the case.

Reply:
Makes sense, thank you for the explanation!

Reply:
public function getIP(): string {
    return ($this->exists() && !$this->isLoggedIn() ? $this->data()->ip : Util::getRemoteIp())
}
???